parser.y

```diff
@@ -12015,4 +12060,31 @@ EncryptionOpt:
 		}
 		$$ = $1
 	}
+
+ValuesStmtList:
+	RowStmt
+	{
+		$$ = append([]*ast.RowExpr{}, $1.(*ast.RowExpr))
+	}
+|	ValuesStmtList ',' RowStmt
+	{
+		$$ = append($1.([]*ast.RowExpr), $3.(*ast.RowExpr))
+	}
+
+RowStmt:
+	"ROW" RowStmtField %prec higherThanComma
+	{
+		$$ = $2
+	}
+
+RowStmtField:
+	RowValue
+	{
+		$$ = &ast.RowExpr{Values: $1.([]ast.ExprNode)}
+	}
+|	RowStmtField ',' RowValue
+	{
+		values := append($1.([]ast.ExprNode), $3.(ast.ExprNode))
+		$$ = &ast.RowExpr{Values: values}
+	}
```

Comment:
This rule seems to be unused. We can remove `RowStmtField` and use `RowValue` instead.